js/deer-record.js

```diff
@@ -226,7 +227,7 @@ export default class DeerReport {
     processRecord(event) {
         event.preventDefault()
         this.evidence = this.elem.getAttribute(DEER.EVIDENCE) // inherited to inputs
-        this.context = this.elem.getAttribute(DEER.CONTEXT) // inherited to inputs
+        this.context = this.elem.getAttribute(DEER.CONTEXT) ? this.elem.getAttribute(DEER.CONTEXT) : "" 
```

Comment:
I think we should let this be null. I would also leave the comment, since this is still inherited to inputs.

Reply:
I am OK with that.